deltachat-rpc-client/examples/echobot.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+import asyncio
+import logging
+import sys
+
+import deltachat_rpc_client as dc
+
+
+async def main():
+    rpc = await dc.start_rpc_server()
+    deltachat = dc.Deltachat(rpc)
+    system_info = await deltachat.get_system_info()
+    logging.info("Running deltachat core %s", system_info["deltachat_core_version"])
+
+    accounts = await deltachat.get_all_accounts()
+    account = accounts[0] if accounts else await deltachat.add_account()
+
+    await account.set_config("bot", "1")
+    if not await account.is_configured():
+        logging.info("Account is not configured, configuring")
+        await account.set_config("addr", sys.argv[1])
+        await account.set_config("mail_pw", sys.argv[2])
+        await account.configure()
+        logging.info("Configured")
+    else:
+        logging.info("Account is already configured")
+        await deltachat.start_io()
+
+    async def process_messages():
+        fresh_messages = await account.get_fresh_messages()
+        fresh_message_snapshot_tasks = [
+            message.get_snapshot() for message in fresh_messages
+        ]
+        fresh_message_snapshots = await asyncio.gather(*fresh_message_snapshot_tasks)
+        for snapshot in reversed(fresh_message_snapshots):
+            if not snapshot.is_info:
+                await snapshot.chat.send_text(snapshot.text)
+            await snapshot.message.mark_seen()
+
+    # Process old messages.
+    await process_messages()
```

Comment:
what happens if you had lets say 1000 "old"/fresh messages but after calling `get_fresh_messages()` while you are processing them 20 new fresh messages arrived, but you where not listening to events yet, will `account.wait_for_event()` in the following lines start giving that messages 20 messages or only start giving new messages that arrive from the point you starting waiting for event?

Reply:
nice!